tools/awps-tunnel/client/src/components/api/Path.tsx

```diff
@@ -0,0 +1,17 @@
+import {PathItem} from "../../models";
+import {Method} from "./Methods";
+
+export function Path({pathItem, path, methodName}: {
+	pathItem: PathItem,
+	path: string,
+	methodName: string
+}): React.JSX.Element {
+	return (<div style={{flex: 4}}>
+		<div>
+			{pathItem.post && methodName === "post" && <Method method={pathItem.post} path={path} methodName={methodName}/>}
```

Comment:
if this enough: `<Method method={pathItem[methodName]} path={path} methodName={methodName}`/>

Reply:
Type 'Operation | undefined' is not assignable to type 'Operation'.
Type 'undefined' is not assignable to type 'Operation'.ts(2322)

Reply:
reference: models.ts line 127 - line 136